### PR TITLE
Update issue templates

### LINKED ISSUE #3: Create Issue templates
Create two issue templates, one each for a bug report and feature request.

### PR DESCRIPTION
**Pull request by SaurabhAgarwala**
Fixes #3 by @SaurabhAgarwala 

This PR adds two issue templates, one each for a bug report and feature request

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug Report
+about: Create a report to help us improve
```
Comment (kartikcho) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Let's change this to "Report a bug you've encountered in the project"

**Review by kartikcho — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem else remove this section.
+
+**Desktop (please complete the following information else remove this section.):**
+ - OS: [e.g. iOS]
```
Comment (smartclash) [inline .github/ISSUE_TEMPLATE/bug-report.md]: iOS is not a desktop operating system

```suggestion
 - OS: [e.g. macOS, Windows, Linux]
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information else remove this section.):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
```
Comment (smartclash) [inline .github/ISSUE_TEMPLATE/bug-report.md]: We shouldn't ask them to just say "stock browser". Let them tell what browser they used specifically.

```suggestion
 - Browser [e.g. safari, firefox, chrome]
```

**Review by smartclash — CHANGES_REQUESTED** [review]
I've suggested some changes, so you can just commit them in.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (SaurabhAgarwala) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Oops! Yups corrected it

**Comment by SaurabhAgarwala** [comment]
Implemented both of yours suggested changes in a single commit. Thanks for reviewing

**Review by smartclash — APPROVED** [review]
Yeah, all good :)

**Review by kartikcho — APPROVED** [review]
Looks good to me, let's get this merged!

**Pull request merged**